lesson1/src/stacks_queues/queue.js

```diff
@@ -12,17 +12,26 @@ class Queue {
   // Dequeue should dequeue an element from the queue
   dequeue() {
     // Todo: add your code here !
+    let popped = this.queue[0];
+    this.queue.shift();
```

Comment:
yes this works... and yes im not gonna say anything about using `.shift()` ... but i would RALLY like to know that when you do something like this you are still conscious about the order of your algorithm. Whats the order of your dequeue? it now depends on shift... which seems O(n) right? we know we can do better 😉  but if you just wanted to finish it that's also great ! but next time leave a comment like `// my dequeue is now O(n) 😞` so at least i know you made a conscious decision !  (sorry about the rant ill say this in class next time !)